apps/api-documenter/src/documenters/ExperimentYamlDocumenter.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { PackageName } from '@microsoft/node-core-library';
+import { DocComment, DocInlineTag } from '@microsoft/tsdoc';
+import { ApiModel, ApiItem, ApiItemKind, ApiDocumentedItem } from '@microsoft/api-extractor-model';
+
+import { IConfigFile, IConfigTableOfContents } from './IConfigFile';
+import { IYamlTocItem, IYamlTocFile } from '../yaml/IYamlTocFile';
+import { YamlDocumenter } from './YamlDocumenter';
+
+/**
+ * EXPERIMENTAL - This documenter is a prototype of a new config file driven mode of operation for
+ * API Documenter.  It is not ready for general usage yet.  Its design may change in the future.
+ */
+export class ExperimentYamlDocumenter extends YamlDocumenter {
```

Comment:
>ExperimentYamlDocumenter [](start = 13, length = 24)

Rename this class to `ExperimentalYamlDocumenter`

Reply:
Renamed